src/features/world/ui/megastore/seasonalstore_components/ItemDetail.tsx

```diff
@@ -427,7 +426,7 @@ export const ItemDetail: React.FC<ItemOverlayProps> = ({
                       </Label>
                     )}
 
-                    {!isTradeable && (
+                    {(!isTradeable || isTradeable === undefined) && (
```

Comment:
This explicit `undefined` check is redundant. It would be covered by `!isTradeable`.